integration/bench/stress_test_write_path.js

```diff
@@ -15,24 +15,20 @@ export let options = {
       exec: 'writePath',
       startVUs: 1,
       stages: [
-        { duration: '10m', target: 5 },
-        { duration: '7m', target: 10 },
-        { duration: '4m', target: 15 },
-        { duration: '1m', target: 20 },
-        { duration: '5m', target: 0 },
+        { duration: '1m', target: 5 },
```

Comment:
(Nit/idea?)

I used scenarios for the scripts, and well there are pretty recent! So there is no good way to override these values (stages, durations, VUs) from the CLI.

But, I still think it's beneficial to have the stairway stages here, in case someone wants to run it on their Tempo deployment. Maybe we can add a CI env variable? And switch configs depending on that.

Another way is to do something like this: `--env scenarios={<some json>}` and `JSON.parse(__ENV.scenarios)`

Reply:
Yeah, I felt bad removing these stages as it's a pretty clever feature.  I will think about a way to make them configurable or restore some of this functionality.

Probably not parsing json out of an env var though.  That sounds like a lot of nasty escaping in bash :)

Reply:
Agree about the JSON parsing :smile: 

Maybe having a CI env var is the easiest. If the env var exists, use some config, if not use the other one.

Reply:
Readded some stages and extended the test to 5 minutes.  I like your long term "soak" test, but don't think we can include it in CI.